scripts/kibana/validate-ts-interfaces-against-apm-server-sample-docs/scripts/clone-kibana.sh

```diff
@@ -1,9 +1,12 @@
+#!/usr/bin/env bash
+set -e
+
 OWNER=${1:-elastic}
 BRANCH=${2:-master}
 
-echo "Cloning Kibana: $OWNER:$BRANCH"
+echo "Cloning Kibana: ${OWNER}:${BRANCH}"
 
-cd ./tmp
-git clone --depth 1 -b $BRANCH https://github.com/$OWNER/kibana.git
-mv ./kibana/x-pack/legacy/plugins/apm/typings/es_schemas ./apm-ui-interfaces
+cd ./tmp || true
+git clone --quiet --depth 1 -b "${BRANCH}" "https://github.com/${OWNER}/kibana.git"
+mv ./kibana/x-pack/plugins/apm/typings/es_schemas ./apm-ui-interfaces
```

Comment:
The real change!
Question:
- Do we need to support backward compatibility?

if so

```bash
newLocation=./kibana/x-pack/plugins/apm/typings/es_schemas
location=./kibana/x-pack/legacy/plugins/apm/typings/es_schemas
if [ -d "${newLocation}" ] ; then
   location=${newLocation}
fi
mv "${location} ./apm-ui-interfaces
```

Reply:
Good question. If the above works I think that's a good approach. Perhaps add a comment saying that in 7.7 files moved around. Then we can move this again in a couple of minors from now.